src/styles/five-stars-review.css

```
@@ -0,0 +1,30 @@
+/* #region five-stars-review*/
+
+.five-stars__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.five-stars__item {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  list-style-type: none;
+  background-position: center;
+}
+
+.five-stars__item:not(:last-child) {
+  margin-right: 4px;
+}
+
+.five-stars__list--4 > li.five-stars__item:nth-of-type(-n+4) {
```

```suggestion
.five-stars__list--4 > :nth-of-type(-n+4) {
```
Тут вот так тоже доджно работать